src/jit/instr.cpp

```diff
@@ -1910,6 +1911,7 @@ regNumber CodeGen::genGetZeroRegister()
 
     return zeroReg;
 }
+#endif
```

Comment:
nit: remove the nested `#ifdef LEGACY_BACKEND`. And since you're removing the `REDUNDANT_LOAD`, maybe remove that one, too.

Reply:
I tried defining LATE_DISASM and it gets 11 unresolved symbols all related to DIS.
I suppose it I actually tried linking the disassembler library it work work.

Reply:
And the ifdef I added can be merged with the one from the next function (inst_TT_IV)

Reply:
Done